src/main/java/frc/robot/commands/intakeCommands/RunIntakeCommand.java

```diff
@@ -69,28 +65,34 @@ public boolean isFinished() {
 
       if (coralIntaken) {
         Robot.rumble().coralIntaken.schedule();
-        if (boathook.getLevel() == Level.L1) {
-          liftToL1.schedule();
-          intake.flipL1Toggle();
-        } else {
-          runHandoff.schedule();
+        if (!runningWait.isRunning()) {
+          runningWait.restart();
+          return false;
         }
-      }
 
-      return coralIntaken;
+        if (runningWait.hasElapsed(.2)) {
+          if (boathook.getLevel() == Level.L1) {
```

Comment:
TBH we probably shouldn't be having side effects in isFinished. Apologies for introducing that pattern.